gui/src/app/state/receive.rs

```diff
@@ -154,12 +155,20 @@ impl State for ReceivePanel {
                 ));
                 Command::none()
             }
+            Message::View(view::Message::ShowQrCode(i)) => {
+                let address = self.addresses.list.get(i).expect("Must be present").clone();
```

Comment:
It is unexpected to happen, but it is safer to do the `if let Some(address) = self.address.get(i).as_ref()`
(I think `as_ref` will help us to not use `.clone()`)